api/orders.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const router = express.Router();
+const {
+  getAllOrders,
+  getOrderById,
+  createOrder,
+  updateOrder,
+  addProductToOrder,
+  destroyOrder,
+} = require("../db");
+const { requireUser } = require("./utils");
+
+router.get("/", async (req, res, next) => {
+  const orders = await getAllOrders();
+
+  res.send(orders);
+});
+
+router.post("/", requireUser, async (req, res) => {
+  const { subtotal } = req.body;
+  const orderData = {};
+
+  try {
+    orderData.creatorId = req.user.id;
+    orderData.subtotal = subtotal;
+
+    const order = await createOrder(orderData);
+
+    if (!order) {
+      next({
+        name: "UserNotAuthorized",
+        message: "User is not authorized to create an order.",
+      });
+    }
```

Comment:
change error message: something went wrong creating your order? im not sure what a good message here is

Reply:
I changed it. Does that work?